worlds/sc2/client.py

```diff
@@ -1094,6 +1094,7 @@ async def on_step(self, iteration: int):
             kerrigan_options = calculate_kerrigan_options(self.ctx)
             soa_options = caclulate_soa_options(self.ctx)
             uncollected_objectives: typing.List[int] = self.get_uncollected_objectives()
+            mission_variant = 0  # temp value until generator logic is done
```

Comment:
Add a TODO comment, also mention race-swapping there with its values, as currently understood by the mod